tests/e2e/test_rosacli_node_pool.go

```diff
@@ -1777,4 +1777,100 @@ var _ = Describe("Edit nodepool",
 							"ERR: Expected a valid http tokens value : ec2-metadata-http-tokens " +
 								"value should be one of 'required', 'optional'"))
 			})
+
+		It("Edit machine pool to the ROSA Hypershift cluster - [id:56778]", labels.Runtime.Day2, labels.High,
+			func() {
+				var (
+					replicas         string
+					machinepoolNames = []string{"np-56778", "np-56778-1"}
+				)
+
+				for _, npName := range machinepoolNames {
+					By("List Machinepools")
+					machinePoolList, err := rosaClient.MachinePool.ListAndReflectNodePools(clusterID)
+					Expect(err).ToNot(HaveOccurred())
+					np := machinePoolList.Nodepool("workers-0")
```

Comment:
This won't work on single AZ cluster

Reply:
fixed it, now it will work for both single as well as multi az clusters